src/executor/select.rs

```diff
@@ -997,6 +998,27 @@ where
     L: EntityTrait,
     R: EntityTrait,
 {
+    // #[cfg(feature = "hashable-value")]
```

Comment:
I think sea_query::Value is not hashable by default and a specified feature is needed 
but then the tests would need to also have the feature on in the configuration
my thought for solution is to make sea_query::Value to be hashable by default

Reply:
Yes the 'hashable-value' feature is intended to be add by default.